visidata/shell.py

```diff
@@ -263,6 +263,8 @@ def inputShell(vd):
 DirSheet.addCommand('y', 'copy-row', 'copy_files([cursorRow], inputPath("copy to dest: "))', 'copy file to given directory *path*')
 DirSheet.addCommand('gy', 'copy-selected', 'copy_files(selectedRows, inputPath("copy to dest: ", value=cursorRow.given))', 'copy selected files to given directory *path*')
 
+DirSheet.addCommand('zo', 'open-cell-file', 'ft = input("filetype: ", type="filetype", value=options.filetype or Path(cursorDisplay).ext); vd.push(openSource(cursorDisplay, filetype=ft) or fail(f"file {cursorDisplay} does not exist"))')
```

Comment:
If this gets cumbersome or used elsewhere we can pull it off into `inputFiletype` like others we have.  No need for now though.